inspector/src/main/java/org/apache/servicecomb/inspector/internal/InspectorConfig.java

```diff
@@ -31,15 +33,17 @@ public boolean isEnabled() {
     return enabled;
   }
 
-  public void setEnabled(boolean enabled) {
+  public InspectorConfig setEnabled(boolean enabled) {
```

Comment:
Can you add this usage for this feature to demo / integration tests? so that we can add documents using the demo.

Reply:
it's a shortcut for https://docs.servicecomb.io/java-chassis/zh_CN/config/read-config/  
declare a class with @InjectProperties to be a bean, that's enough.